src/Security/perf/Microbenchmarks/Microsoft.AspNetCore.Security.Microbenchmarks.csproj

```diff
@@ -14,6 +14,7 @@
     <Reference Include="Microsoft.AspNetCore.Http" />
     <Reference Include="Microsoft.Extensions.DependencyInjection" />
     <Reference Include="Microsoft.Extensions.Logging" />
+    <Reference Include="Microsoft.AspNetCore.Authentication.Cookies" />
```

Comment:
Nit: We tend to sort `@(Reference)` items which would place this second, after the already-out-of-place `BenchmarkDotNet` reference. Up to you whether to include this in your next iteration.

Reply:
Thanks, it was resolved